syntaxes/examples/gdscript2.gd

```diff
@@ -0,0 +1,55 @@
+extends Node
```

Comment:
I think it'd be good idea to submit this code as a GDScript integration test in godotengine/godot :slightly_smiling_face: 

I can look into doing that.

Reply:
Is there an ergonomic way to automate a highlighting test? I haven't been able to think of one that wouldn't be a massive pain to define the success/failure conditions.

I think finding a way to share test files with the engine is a great idea, though. Having files that we know exercise all of the compiler's features would really increase confidence that we're covering all the possible syntax.